web3/providers/eth_tester/provider.py

```diff
@@ -115,6 +126,30 @@ def create_log_filter(eth_tester, params):
     return filter_id
 
 
+def generate_random_private_key():
+    return encode_hex(bytes(bytearray((
+        random.randint(0, 255)
```

Comment:
Maybe a little note here in case anyone stumbles in, saying that this shouldn't be used for generating private keys in production, because `random` is not meant for generating secure secrets.